algorithms/ar-msrt/javascript/merge_sort.js

```diff
@@ -0,0 +1,36 @@
+function mergeSort(array) {
+    const merge = function (array1, array2) {
+        if (array1 == [] || array1 == undefined) {
+            return array2;
+        }
+        else if (array2 == [] || array2 == undefined) {
+            return array1;
+        }
+        else {
+            let output = [];
+            while (array1.length > 0 || array2.length > 0) {
+                if (array1.length == 0) {
```

Comment:
Codacy found an issue: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6334421)